tests/Oryx.BuildImage.Tests/Node/NodeJsSampleAppsOtherTests.cs

```diff
@@ -976,6 +976,9 @@ public void  CanBuildAndRunNodeApp_UsingYarn2ForBuild()
             var appOutputDir = "/tmp/nextjs-yarn2-example";
             var appDir = volume.ContainerDir;
             var script = new ShellScriptBuilder()
+                .AddCommand($"yarn set version berry")
```

Comment:
we needed to set the workspace one time, this should  be user's responsibility, so adding these commands in test prior to oryx build command.
Assumption is User will run these commands in his/her local env before checking it in (including workspace) in git